dipy/viz/projections.py

```diff
@@ -13,7 +13,7 @@
 
 matplotlib, has_mpl, setup_module = optional_package("matplotlib")
 plt, _, _ = optional_package("matplotlib.pyplot")
-tri, _, _ = optional_package("matplotlib.tri")
+mpl_tri, _, _ = optional_package("matplotlib.tri")
```

Comment:
is it use ? should we just remove it ?  I do not find any use of `mpl_tri`.

Reply:
I know Serge, but I imagined that the package needs to be imported so that some sort of processing on the mesh can be completed. IIRC sometimes with `matplotlib` this happens, but I cannot point to an example out of my mind. Best would probably be to have a test and see if removing it makes any difference. Can be done in a separate PR.

Reply:
ok, let's check that in a different PR. thank you for the feedback. merging